smart-contracts/contracts/cl-vault/src/vault/swap.rs

```diff
@@ -85,6 +76,7 @@ pub fn execute_swap_non_vault_funds(
             vault_config.swap_max_slippage.denominator(),
         )?;
 
+        // Add swap messages
```

Comment:
this comment does not provide more information than the following code, plz remove